geo/liblwgeom/lwgeom_geos_cluster.cpp

```diff
@@ -37,8 +37,8 @@ static const int STRTREE_NODE_CAPACITY = 10;
 /* Utility struct used to accumulate items in GEOSSTRtree_query callback */
 struct QueryContext {
 	void **items_found;
-	uint32_t items_found_size;
 	uint32_t num_items_found;
+	uint32_t items_found_size;
```

Comment:
Why does this need to change?

Reply:
The compiler raise an exception about the order of arguments for this line :

https://github.com/handstuyennn/geo/blob/15ad5a691f55bbe263b4b5d1bf710cfe1fac09b3/geo/liblwgeom/lwgeom_geos_cluster.cpp#L187-L202

Reply:
Thanks